lib/base/Modeler.js

```diff
@@ -19,40 +19,26 @@ import {
   DmnPropertiesProviderModule
 } from 'dmn-js-properties-panel';
 
-import { createOverviewModule } from './features/overview/Overview';
-
 export default class CamundaDmnModeler extends DmnModeler {
 
   constructor(options = {}) {
 
     const {
       common = {},
-      moddleExtensions = {},
```

Comment:
Why did we need this one in the first place?

Reply:
I believe we didn't.